tests/integration/devfile/cmd_devfile_watch_test.go

```diff
@@ -86,4 +96,44 @@ var _ = Describe("odo devfile watch command tests", func() {
 			Expect(output).To(ContainSubstring("Error: unknown flag: --devfile"))
 		})
 	})
+
+	Context("when executing odo watch after odo push", func() {
+		It("should listen for file changes", func() {
+			// Devfile push requires experimental mode to be set
+			helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
```

Comment:
We should move this to AfterEach()

Reply:
I did initially have it in `BeforeEach` but a test in watch, check for non-experimental mode  `when executing odo watch with devfile flag without experimental mode`.

I'll move the set experimental true to `BeforeEach` and reset to false for the above test